rdt/transformers/numerical.py

```diff
@@ -82,7 +82,7 @@ def _learn_rounding_digits(data):
                 if (roundable_data == roundable_data.round(decimal)).all():
                     return decimal
 
-        else:
+        elif len(roundable_data) > 0:
```

Comment:
I assume `elif roundable_data` doesn't work because it still creates a series with no rows right?

Reply:
Yeah it's an empty series